### DISCUSSION TITLE
How to set the port for ready and health probes?

### DISCUSSION THREAD

**Discussion by edocrippaofficial** [mia-platform/community#312] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.6.0

### Description

Hi, is there a way to set the port for the k8s probes?   
I want my probes to contact the 8080 port of my pod, but every time I save the configuration the deployment file is overwritten with the standard 3000 value on the probes ports.

I can't find any info in the [doc section](https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/microservice-runtime-resources#probes)

Thanks

**Comment by edocrippaofficial** [comment, marked as answer]
Found it: the console uses the HTTP_PORT env to generate the probes spec

**Comment by peppemanzi** [comment]
Have you tried to use the HTTP_PORT variable? The page you have linked says that it is used for TCP socket check (https://docs.mia-platform.eu/docs/development_suite/api-console/api-design/microservice-runtime-resources#path). I think it is the same also for the endpoints case.

**Reply by edocrippaofficial** [reply]
Yes @peppemanzi , I too found that that env is the one responsible for the probes port...
We can specify that more clearly in the doc, what do you think?

**Reply by peppemanzi** [reply]
For sure! Feel free to open a PR on https://github.com/mia-platform/documentation
